features/ubuntu_pro.feature

```diff
@@ -279,6 +289,16 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
             ros           +no  +(-|—) +Security Updates for the Robot Operating System
             ros-updates   +no  +(-|—) +All Updates for the Robot Operating System
             """
+        When I run `systemctl start ua-auto-attach.service` with sudo
+        And I verify that running `systemctl status ua-auto-attach.service` `as non-root` exits `0,3`
```

Comment:
Shouldn't this always exit 0 ?

Reply:
No. The service is not always running. If the service ran successfully but now is dead, the return code is 3.